dags/readme.md

```diff
@@ -16,47 +16,70 @@ Our instance of Airflow runs as its own user `airflow` which resides in `/etc/ai
 
 ## Common Tasks
 
-### Creating a new airflow DAG
+### Creating a new Airflow DAG
 
-1) Branch off `master` and do your work in your normal git folder.
+1) Branch off `master` and do your work in your normal git folder, like you do.
 
 2) Create your dag in the dag folder and if applicable, your related script in another appropriate folder.
 
 3) Commit and push to GitHub.
 
-4) pull your testing branch into `dev_scripts`, **DO NOT PUSH ANYTHING IN THIS BRANCH**.
+4) Navigate to `/etc/airflow/dev_scripts` and pull your testing branch into `dev_scripts` using: 
 
-4) Test your dag by added a new symbolic link using the following (make sure you are in the correct directory `/etc/airflow/dag` ):
+```
+git pull origin your_branch_name
+```
+
+**DO NOT PUSH ANYTHING IN THIS BRANCH**.
+
+5) Navigate to `/etc/airflow/dag` 
+
+6) Create a new symbolic link using:
 
 ```
 ln -s /etc/airflow/dev_scripts/dags/your_dag.py your_dag.py
 ```
 
-5) Repeat step 3 until your test is successful.
+7) In a new browser tab, go to https://10.160.2.198/airflow/admin/ and enter your Airflow credentials.
+
+**Note that the IP address is the address for the EC2 server (so if the EC2's IP address changes, the link will need to change too).**
+
+8) Go to the DAGs tab and find your dag by searching or scrolling though the list. Note that the name of the dag in Airflow is the same as the name in the dag variable within the .py script. It's a reallllly good idea to keep these the same, but if they are different, you may experience an unexpected scavenger hunt.
 
-6) Make a pull request to master.
+9) Trigger your dag and watch what happens. Correct any errors shown in the log. Ensure any alerts are working as expected. Check the output where appropriate. Stay on this step until there are no more errors and everything is functioning splendidly.
```

Comment:
Thanks, Sarah. These steps are very clear and easy to follow.
One question: how to trigger a DAG? I think just enabling it from the UI should trigger the most recent run (if the DAG is programmed to run on a schedule not manually), right? 
P.S., in Airflow 2, it's easier to trigger a DAG on specific date from the UI

Reply:
I added some more explicit instructions on how to trigger a DAG. I had to hunt around a bit to find it so it was definitely necessary... thanks for noticing!!!